extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+<!-- omit in toc -->
+### `3DTILES_implicit_tiling` subtree extension
+
+An object assigning tile metadata to implicit tiles in the subtree. Tile metadata only exists for available tiles and is tightly packed by increasing tile index. To access individual tile metadata, implementations may create a mapping from tile indices to tile metadata indices.
+
+**`3DTILES_implicit_tiling subtree extension` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**class**|`string`|The class that tile metadata conforms to. The value must be a class ID declared in the `classes` dictionary.|No|
+|**properties**|`object`|A dictionary, where each key corresponds to a property ID in the class' `properties` dictionary and each value is an object describing where property values are stored. Optional properties may be excluded from this dictionary.|No|
+|**extensions**|`any`||No|
+|**extras**|`any`||No|
+
+Additional properties are allowed.
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.class
+
+The class that tile metadata conforms to. The value must be a class ID declared in the `classes` dictionary.
+
+* **Type**: `string`
+* **Required**: No
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.properties
+
+A dictionary, where each key corresponds to a property ID in the class' `properties` dictionary and each value is an object describing where property values are stored. Optional properties may be excluded from this dictionary.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: `subtree.property`
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.extensions
+
+* **Type**: `any`
+* **Required**: No
+
+<!-- omit in toc -->
+#### 3DTILES_implicit_tiling subtree.extras
+
+* **Type**: `any`
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-subtree-property"></a>
+<!-- omit in toc -->
+### Subtree property
+
+An array of binary property values.
+
+**`Subtree property` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**bufferView**|`integer`|The index of the buffer view containing property values. The data type of property values is determined by the property definition: When `type` is `BOOLEAN` values are packed into a bitfield. When `type` is `STRING` values are stored as byte sequences and decoded as UTF-8 strings. When `type` is a numeric type values are stored as the provided `type`. When `type` is `ENUM` values are stored as the enum's `valueType`. When `type` is `ARRAY` elements are packed tightly together and the data type is based on the `componentType` following the same rules as above. `arrayOffsetBufferView` is required for variable-size arrays and `stringOffsetBufferView` is required for strings (for variable-length arrays of strings, both are required). The buffer view `byteOffset` must be aligned to a multiple of 8 bytes.| &#10003; Yes|
```

Comment:
Somewhere a diagram for these would be valuable even if it needs to be in more than one spec document so the reader isn't expected to have everything memorized or know when to cross reference.